### DISCUSSION TITLE
Access to a new element.

### DISCUSSION THREAD

**Discussion by DobroFenix** [nilsteampassnet/TeamPass#4542] [Q&A] [answered]
I'm creating a new element. In its settings, I specify for whom I grant access to this element.:
![image](https://github.com/user-attachments/assets/56e95b7d-4bfb-46c0-b725-54692a417ca1)

But after its creation, the established powers disappear (are not saved).
![image](https://github.com/user-attachments/assets/40c2be7f-d271-4319-9314-347b20db2c4b)

Until the task of creating a new element is completed, no one can change the element and set the correct permissions for it. And the process of creating a new element is very long.
![image](https://github.com/user-attachments/assets/597e2cd5-826e-484b-98a8-684ec5fb78cb)

Why don't keep access rights to it when creating a new item?

**Comment by DobroFenix** [comment]
HELP

**Comment by corentin-soriano** [comment, marked as answer]
Can you try #4546 please?

**Reply by DobroFenix** [reply]
Thank you so much.
Now the system has begun to work correctly in terms of rights when creating new elements.
I'll go ahead and test TeamPass.

It's strange why it didn't work initially.
